<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1163</width>
    <height>908</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QToolBar {
    spacing: 5px; /* spacing between items in the tool bar */
}

QToolButton:hover {
    background: #bcc7d9;
}

QStatusBar {
    border-width: 5px;
    border-color: blue;
}</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2"/>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <property name="minimumSize">
          <size>
           <width>240</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>240</width>
           <height>16777215</height>
          </size>
         </property>
         <widget class="QGroupBox" name="groupBox">
          <property name="geometry">
           <rect>
            <x>11</x>
            <y>11</y>
            <width>218</width>
            <height>381</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>9</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="title">
           <string>Acquisition</string>
          </property>
          <widget class="QLabel" name="label">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>30</y>
             <width>121</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Autosave Folder Path</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton">
           <property name="geometry">
            <rect>
             <x>140</x>
             <y>140</y>
             <width>71</width>
             <height>28</height>
            </rect>
           </property>
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
          <widget class="QWidget" name="layoutWidget">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>190</y>
             <width>201</width>
             <height>166</height>
            </rect>
           </property>
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>FFT Length</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="comboBox">
              <item>
               <property name="text">
                <string>262144</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Duration</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>FFT Window</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QComboBox" name="comboBox_2">
              <item>
               <property name="text">
                <string>Hanning</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Flat Top</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>None</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>High-pass</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="lineEdit_3">
              <property name="text">
               <string>20</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Low-pass</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLineEdit" name="lineEdit_4">
              <property name="text">
               <string>20000</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Weighting</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QComboBox" name="comboBox_3">
              <item>
               <property name="text">
                <string>None</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>A-wt</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSpinBox" name="spinBox">
              <property name="minimum">
               <number>6</number>
              </property>
              <property name="maximum">
               <number>20</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QTextEdit" name="textEdit">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>50</y>
             <width>201</width>
             <height>81</height>
            </rect>
           </property>
           <property name="html">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'PMingLiU'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;c:\&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </widget>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>400</y>
            <width>221</width>
            <height>151</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="title">
           <string>Advanced Settings</string>
          </property>
          <widget class="QCheckBox" name="checkBox">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>120</y>
             <width>101</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>Mixed Noise</string>
           </property>
          </widget>
          <widget class="QWidget" name="">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>50</y>
             <width>201</width>
             <height>58</height>
            </rect>
           </property>
           <layout class="QFormLayout" name="formLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Deep Learning</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="comboBox_4">
              <item>
               <property name="text">
                <string>CNN</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>DNN</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>2D_CNN</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Recording wav</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="pushButton_2">
              <property name="text">
               <string>Select</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QLabel" name="label_11">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>580</y>
            <width>221</width>
            <height>161</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1163</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenwav"/>
   </widget>
   <widget class="QMenu" name="menuMode">
    <property name="title">
     <string>Run</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionRun"/>
    <addaction name="actionBacktest"/>
   </widget>
   <widget class="QMenu" name="menuSetup">
    <property name="title">
     <string>Setup</string>
    </property>
    <addaction name="actionTraining"/>
    <addaction name="actionpromptsn"/>
    <addaction name="actiontest"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuHardware">
    <property name="title">
     <string>Hardware</string>
    </property>
    <addaction name="actionRefresh"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuMode"/>
   <addaction name="menuSetup"/>
   <addaction name="menuHardware"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>55</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRun"/>
   <addaction name="actionOpenwav"/>
   <addaction name="actionBacktest"/>
   <addaction name="actionTraining"/>
   <addaction name="actionRefresh"/>
   <addaction name="actionAbout"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionRun">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/images/play.svg</normaloff>:/icons/images/play.svg</iconset>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/images/info.svg</normaloff>:/icons/images/info.svg</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/images/hard-drive.svg</normaloff>:/icons/images/hard-drive.svg</iconset>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </action>
  <action name="actionBacktest">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/images/refresh-cw.svg</normaloff>:/icons/images/refresh-cw.svg</iconset>
   </property>
   <property name="text">
    <string>Backtest</string>
   </property>
  </action>
  <action name="actionOpenwav">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/images/file-plus.svg</normaloff>:/icons/images/file-plus.svg</iconset>
   </property>
   <property name="text">
    <string>Openwav</string>
   </property>
  </action>
  <action name="actionTraining">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/images/cpu.svg</normaloff>:/icons/images/cpu.svg</iconset>
   </property>
   <property name="text">
    <string>Training</string>
   </property>
  </action>
  <action name="actionpromptsn">
   <property name="text">
    <string>promptsn</string>
   </property>
  </action>
  <action name="actiontest">
   <property name="text">
    <string>test</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
